### PR TITLE
Add views

### PR DESCRIPTION
**Pull request by pinkmoon25**
### Blog App - Views

- Used methods from models to get data from database
- Add data to controllers from models by using appropriate methods
- Implement Views for each actions of controlllers
- Add a temporary home controller for root view

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1 +1,25 @@
...
+          <span> <%=post.comments_count%> </span>
+           <span> <%=post.likes_count%> </span>
+        </div>
+        <%post.comments.each do |comment|%>
+          <li><p> <span><%=comment.author.name %> :</span> <%=comment.text%>  </p></li>
+        <%end%>
+      </div>
+    </li>
+    <%end%>
+  </ul>
```
Comment (DeliceLydia) [inline app/views/posts/index.html.erb]: - [x] When you open a user post you should see only 3recent posts and a `link` that leads you to a page that has all posts please work on that, currently I am seeing all posts

File: app/views/posts/show.html.erb [hunk 2]
```diff
@@ -1 +1,16 @@
...
+    <div>
+      <span>Comments: <%=@post.comments_count%> </span>
+      <span>Likes: <%=@post.likes_count%> </span> 
+    </div>
+    <p> <%=@post.text%> </p>
+  </div>
+  <ul>
+  <%@post.comments.each do |comment|%>
+    <li><p> <span><%=comment.author.name %> :</span> <%=comment.text%>  </p></li>
+  <%end%>
```
Comment (DeliceLydia) [inline app/views/posts/show.html.erb:14]: - [x] A post should display only 5 recent, currently it is showing all comments please work on that

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @PinkMoon25, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: app/views/posts/index.html.erb (hunk 1 above)
Comment (pinkmoon25) [reply app/views/posts/index.html.erb]: Hi @DeliceLydia , the requirements suggests that we should display recent posts for /user.id path which links to users#how page, and for posts/index.html we are suppose to display all posts and 5 recent comments which i have implemented.

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @PinkMoon25  😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**Highlights**
- Well done on making the required changes
- Great communication

Cheers and Happy coding!👏👏👏

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**